Formula/joe.rb

```diff
@@ -17,6 +17,6 @@ def install
   end
 
   test do
-    system "#{bin}/stringify"
+    assert_match "Joe's Own Editor v#{version}", shell_output("TERM=tty #{bin}/joe -help")
```

Comment:
I didn't try; I figured that the TERM=tty setting was specific to the particular test; not all subsequently-spawned shells.  More a style decision than anything else.  Also, I chose tty, since it'd give the best chance of having no escape sequences in the output.